### ISSUE TITLE
training and validation loss looks funny

### ISSUE THREAD

**Issue by munziliashali** [meijieru/crnn.pytorch#174]
![image](https://user-images.githubusercontent.com/40015034/47557349-a6871000-d93a-11e8-8cf7-dd6cb1427f9f.png)
Dear @meijieru and all,

I am using your model with additional cnn layer on my part to predict my custom dataset and got this funny-looking training loss and validation loss... 
Can you point out how should I fix this?

Thank you.

**Comment by GSWS** [comment]
Hi, did you solve your problem? Did you try using a smaller learning rate?

**Comment by zhengshangdong** [comment]
@munziliashali Hi,if you use batchNormalization=True,the pretrained model can not load  straightly,how you solve this new problem.....Only one way I can come out——rebuild the network and get the convs(weights bias) from pretrained model by keys,but It's too trouble.....

**Comment by Heisenberg0391** [comment]
> @munziliashali Hi,if you use batchNormalization=True,the pretrained model can not load straightly,how you solve this new problem.....Only one way I can come out——rebuild the network and get the convs(weights bias) from pretrained model by keys,but It's too trouble.....

does param 'strict' help? try model.load_state_dict(pretrained_dict, strict=False)
